docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>List activations only for a specific entity, using its <code>entity_id</code>.</p><p> </p><p>The&nbsp;<code>entity_id</code> or <code>EventEntityKind.JOB</code> is formatted as&nbsp;<code>&lt;job-name&gt;.&lt;job_uid&gt;</code>. This is the only field in alert activation that supports wildcard search with asterix <code>*</code>. To enable .....</p><p> </p><p>-----</p><p>Also - what about model endpoints?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
@TomerShor `EventEntityKind.JOB` looks for me more confusing.

Reply:
Alright.
I still think you can add the "intro" line
`List activations only for a specific entity, using its entity_id.`